messages/describe/origin/errors.go

```diff
@@ -0,0 +1,11 @@
+package origin
+
+import "errors"
+
+var (
+	ErrorMissingArguments = errors.New("Required flags are missing. You must supply application-id and origin-id as arguments. Run 'azion <command> <subcommand> --help' command to display more information and try again")
```

Comment:
You can remove this. We never return this error, as we ask for inputs if not sent.